chef.py

```diff
@@ -19,29 +19,32 @@
 from bs4 import BeautifulSoup
 import youtube_dl
 
-from le_utils.constants import content_kinds, file_formats, languages
+from le_utils.constants import content_kinds, file_formats, languages, exercises
 from ricecooker.chefs import SushiChef
-from ricecooker.classes import nodes, files, licenses
+from ricecooker.classes import nodes, files, licenses, questions
 from ricecooker.utils.caching import CacheForeverHeuristic, FileCache, CacheControlAdapter, InvalidatingCacheControlAdapter
 from ricecooker.utils.browser import preview_in_browser
-from ricecooker.utils.html import download_file, WebDriver, minimize_html_css_js
```

Comment:
Yeah, it's annoying that importing `minimize_html_css_js` fails for Py3.4... we'll have to fix that eventually or remove the dependency.